documentation/polymer-templates/tutorial-template-mapped-components-limitations.asciidoc

```diff
@@ -108,6 +108,47 @@ class MainLayout extends LitElement {
 customElements.define("main-layout", MainLayout);
 ----
 
+== Template `disabled` attribute
+
+LitTemplate doesn't support using the `disabled` attribute for an id-mapped component.
```

Comment:
Could perhaps hint what will happens if a component is `disabled` anyway in the template. Does the parser throw an exception or will it silently cause inconsistent state.